app/src/pages/__tests__/FormPageContinueSubmissionAndSave.test.js

```diff
@@ -329,7 +332,7 @@ describe('FormPage continue saved submision then save', () => {
     await waitFor(() => {
       expect(crudDataPoints.saveDataPoint).not.toHaveBeenCalled();
       expect(crudDataPoints.updateDataPoint).toHaveBeenCalledWith({
-        duration: 0,
+        duration: 27235390,
```

Comment:
when we convert this to minutes?

Reply:
actually before call saveDatapoints
https://github.com/akvo/nmis-mobile/blob/feature/108-performance-issue-related-to-datapoint-name/app/src/pages/FormPage.js#L181-L185

let me try to fix the mock test